lib/kb_staging_exporter/Utils/staging_downloader.py

```diff
@@ -409,12 +418,36 @@ def export_to_staging(self, ctx, params):
         
         if not (set(os.listdir(staging_dir)) >= set(files)):
             raise ValueError('Unexpected error occurred during copying files')
+        return files
 
-        returnVal = dict()
-        returnVal['result_dir'] = result_dir
-
-        if generate_report:
-            report_output = self._generate_export_report(files, obj_name, workspace_name)
-            returnVal.update(report_output)
+# TODOS for JSON export:
+# * implement legacy w/ tests
+# * add UI
 
-        return returnVal
+    def export_json_to_staging(self, ctx, params):
+        for p in ['input_ref', 'destination_dir']:
+            if p not in params:
+                raise ValueError('"{}" parameter is required, but missing'.format(p))
+        format = params.get("format")
+        if format == None or format.strip() == "":
+            format = "standard"
+        else:
+            format = format.strip()
+        if format not in ["standard", "legacy_data_import_export"]:
+            raise ValueError("Unknown format: " + format)
+        destination_dir = self._norm_dest_dir(params['destination_dir'])
+        # can't use DFU because it doesn't include provenance
+        obj = self.ws.get_objects2({"objects": [{"ref": params["input_ref"]}]})['data'][0]
+        filename = obj["info"][1].replace("|", "_") + ".json"
+        result_dir = Path(self.scratch, str(uuid.uuid4()))
+        os.makedirs(result_dir, exist_ok=True)
+        if format == "legacy_data_import_export":
+            # TODO JSONDL implement legacy format
+            writeobj = None
```

Comment:
```suggestion
            writeobjs = {}
```

Reply:
This gets replaced in the next PR, it's fine if it fails for now

Reply:
👍